spec/integrations.yml

```diff
@@ -88,6 +88,8 @@
     - optional/capi/string_spec.rb
     '#<RegexpError: invalid multibyte escape: /\xAA/>':
     - language/regexp/escapes_spec.rb
+    "#<Regexp::Scanner::PrematureEndError: Premature end of pattern at #{str}>":
+    - language/regexp/interpolation_spec.rb
```

Comment:
I'd suggest indenting this underneath the string, assuming it is still valid YAML syntax.